# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/ScaledFontBase.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/2d/ScaledFontBase.cpp
--- a/gfx/2d/ScaledFontBase.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/2d/ScaledFontBase.cpp	Sun May 26 14:30:14 2019 +0000
@@ -6,7 +6,7 @@
 
 #include "ScaledFontBase.h"
 
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 
 #ifdef USE_SKIA
 #  include "PathSkia.h"
@@ -36,7 +36,7 @@
 ScaledFont::~ScaledFont() { sDeletionCounter++; }
 
 AntialiasMode ScaledFont::GetDefaultAAMode() {
-  if (gfxPrefs::DisableAllTextAA()) {
+  if (StaticPrefs::DisableAllTextAA()) {
     return AntialiasMode::NONE;
   }
 